target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8064.dtsi

```diff
@@ -863,7 +914,7 @@
 			status = "disabled";
 		};
 
-		sata@29000000 {
+		sata: sata@29000000 {
```

Comment:
It would make sense to add sata and nand changes into separate commit as it makes no sense in `add missing gpio and gsbi declaration` commit.